WalletWasabi.Gui/CrashReporter/Helpers/StartCrashReporterHelper.cs

```diff
@@ -0,0 +1,45 @@
+using System.Reflection;
+using Avalonia;
+using System;
+using WalletWasabi.Logging;
+using Newtonsoft.Json;
+using WalletWasabi.Gui.CrashReporter.Models;
+using System.Diagnostics;
+
+namespace WalletWasabi.Gui.CrashReporter.Helpers
+{
+	public static class StartCrashReporterHelper
```

Comment:
How about having a not static class here and move everything related into this class.

Reply:
A rename would also be good, like CrashReporter. Namespace to WalletWasabi.Gui.CrashReport.

Reply:
If it will be a class there should be some test cases for it.